extension_registry_manager.py

```
@@ -0,0 +1,1108 @@
+                    raise
+
+        except Exception as e:
+            import_results['errors'].append(f"Import failed: {e}")
+            self.logger.error(f"Failed to import extension registry: {e}")
+
+        return import_results
+    
+    def _load_import_data(self, file_path: str, format: str) -> Dict[str, Any]:
+        """Load import data from file."""
+        if format.lower() != 'json':
+            raise ValueError(f"Unsupported import format: {format}. Only 'json' is currently supported.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)